client/src/app/site/pages/meetings/pages/interaction/services/applause.service.ts

```diff
@@ -128,7 +129,7 @@ export class ApplauseService extends BaseICCGatewayService<Applause> {
         if (!applauseLevel) {
             return 0;
         }
-        const quote = applauseLevel / this.maxApplause || 0;
+        const quote = applauseLevel / (this.maxApplause ? this.maxApplause : this.presentApplauseUsers ?? 0) || 0;
```

Comment:
```suggestion
        const quote = applauseLevel / (this.maxApplause ?? this.presentApplauseUsers) || 0;
```
You can skip the second nullish coalescing operator since if `presentApplauseUsers` is nullish, the result will already be `NaN` and `0` will be returned.